SensusService/Protocol.cs

```diff
@@ -1012,6 +1121,50 @@ public void Start()
             }
         }
 
+        public void ScheduleStart()
+        {
+            DateTime startTime = new DateTime(_startTime.Year, _startTime.Month, _startTime.Day, _startTime.Hour, _startTime.Minute, 0);
```

Comment:
When the user specifies a time using the time picker, do they specify seconds? If so, then we should use it. If not, then it's probably zero by default. In either case, we can directly use their specified time in our calculations, obviating the need to reconstruct the variable.

Right?